src/editor/components/modals/revision-diff.jsx

```diff
@@ -50,7 +98,13 @@ export function mapStateToProps(state) {
 
   let currentRevisionContent;
   let previousRevisionContent;
+  let revisionDate;
   if (selectedRevisionId !== undefined && revisionContent) {
+    for (const revision of revisionList) {
+      if (revision.id === selectedRevisionId) {
+        revisionDate = revision.created;
+      }
+    }
```

Comment:
I think we'd be best off just keeping a reference to the selected revision here, that way we can get the needed properties (like `id` or `created`) where needed. In this case, you can just use find to get it:

```js
const selectedRevision = revisionList.find(r => r.id === selectedRevisionId);
```

Reply:
Much better for further cases